From: Lars Kellogg-Stedman <lars@oddbit.com>
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sat, 20 Aug 2022 08:37:28 -0400
Subject: [PATCH 8/8] Remove pidfile and argsfile

Under Alpine Linux, OpenRC manages a pid file on its own; it's not
necessary for slapd to manage one separately. This commit removes the
pidfile and argsfile directives from the example configuration.

diff --git a/servers/slapd/slapd.conf b/servers/slapd/slapd.conf
--- a/servers/slapd/slapd.conf
+++ b/servers/slapd/slapd.conf
@@ -10,9 +10,6 @@
 # service AND an understanding of referrals.
 #referral	ldap://root.openldap.org
 
-pidfile		%LOCALSTATEDIR%/run/slapd.pid
-argsfile	%LOCALSTATEDIR%/run/slapd.args
-
 # Load dynamic backend modules:
 modulepath	%MODULEDIR%
 moduleload	back_mdb.so
diff --git a/servers/slapd/slapd.ldif b/servers/slapd/slapd.ldif
--- a/servers/slapd/slapd.ldif
+++ b/servers/slapd/slapd.ldif
@@ -9,9 +9,6 @@
 #
 # Define global ACLs to disable default read access.
 #
-olcArgsFile: %LOCALSTATEDIR%/run/slapd.args
-olcPidFile: %LOCALSTATEDIR%/run/slapd.pid
-#
 # Do not enable referrals until AFTER you have a working directory
 # service AND an understanding of referrals.
 #olcReferral:	ldap://root.openldap.org
